README-DEVELOP.md

```diff
@@ -10,8 +10,17 @@
 - `logo.png` - a logo for the extension in the marketplace
 - `package.json` - the manifest file declaring all capabilities of the extension
 - `README.md` - for users
+- `tsconfig.json` - configuration for typescript compilation for the client and server code
 - this file for the extension's developers
 
+### The `client` folder
+
+This folder contains the extension code that will be run on the client. it will start and connect to the server defined in `/server`. The client features can be tested by running `npm run test` from the workspace root. Features provided by the client:
```

Comment:
```suggestion
This folder contains the extension code that will be run on the client. It will start and connect to the server defined in `/server`. The client features can be tested by running `npm run test` from the workspace root. Features provided by the client:
```